src/features/navigator/common.ts

```diff
@@ -1,10 +1,39 @@
-import { Cell, NarrativeDoc } from '../../common/types/NarrativeDoc';
+import { Cell } from '../../common/types/NarrativeDoc';
 import { generatePathWithSearchParams } from '../../features/params/paramsSlice';
-import { MockParams } from 'jest-fetch-mock';
 
-export const narrativeSelectedPath = '/narratives/:id/:obj/:ver';
-export const narrativeSelectedPathWithCategory =
-  '/narratives/:category/:id/:obj/:ver';
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+export const emptyFunction = () => {};
+
+// Narrative and Navigator paths
+export const narrativeURL = (wsId: number | string) =>
+  `https://ci.kbase.us/narrative/${wsId}`;
```

Comment:
We probably don't want to hardcore the domain here but we'd probably need a build config option to make it configurable. Ok with this for now.

Reply:
Agreed, the config requirements are being tracked in [URO-142](https://kbase-jira.atlassian.net/browse/URO-142).